OCP-Infrastructure/helm/ToDos.md

```diff
@@ -0,0 +1,7 @@
+# ToDos for HELM-ifying Hogarama
```

Comment:
It seems in this merge request there are two aspects mixed together. OCP4 and helm.

Reply:
The focus is of this merge request ist to providing a deployment mechanism in openshift that reflects our entire infrastructure. As such we have first created bash scripts that can recreate the entire infrastructure, added in Operators to reduce the number of infrastructure resources that we need to maintain and move the remaining infrastructure resources to helm.